tests/with_image_deps/BUILD.bazel

```diff
@@ -5,24 +5,34 @@ load("//tests:test_defs.bzl", "helm_package_regex_test")
 
 exports_files(["Chart.lock"])
 
+EXCLUDE_WINDOWS = select({
```

Comment:
Note that integration tests with rules_oci is currently disabled due to https://github.com/abrisco/rules_helm/issues/53